pkg/controller/watch.go

```diff
@@ -230,6 +230,9 @@ func (w *watch) Start(ctx context.Context) error {
 			if !errors.IsNotFound(err) {
 				w.log.Errorf("loading configmap %s/%s failed: %s", common.NamespaceKubeSystem, common.NameGardenerShootInfo, err)
 				continue
+			} else {
+				// NOTE: client-go returns an empty object, even if the requested object does not exist.
+				shootInfo = nil
```

Comment:
golangci-lint check is failing:
superfluous-else: if block ends with a continue statement, so drop this else and outdent its block

Reply:
fixed 🙂